engines/nvme.h

```diff
@@ -7,6 +7,8 @@
 #ifndef FIO_NVME_H
 #define FIO_NVME_H
 
+#define __SANE_USERSPACE_TYPES__
```

Comment:
This should probably go in the Makefile for mips64, not in this specific location?